setup.py

```diff
@@ -109,6 +109,8 @@
         "pandas",
         # Plotting learning curves
         "matplotlib",
+        # For hooks on recurrent policy
+        "mamba_lens @ git+https://github.com/Phylliida/MambaLens.git",
```

Comment:
That looks good. I think making CI pass probably requires an updated Docker image.